requirements.txt

```diff
@@ -1,25 +1,24 @@
 MarkupPy==1.14
 pyyaml==5.3.1
-django==2.2.10
-django-colorfield==0.1.15
-django-extensions==2.2.5
-django-import-export==1.2.0
-django-leaflet==0.25.0
-django-rq==2.1.0
-jinja2==2.10.3
-markdown2==2.3.8
+django==3.1
+django-colorfield==0.3.2
+django-import-export==2.4
+django-leaflet==0.27.1
+django-rq==2.4.0
+# jinja2==2.10.3
+markdown2==2.3.10
 odfpy==1.4.1
-openpyxl==3.0.3
-pillow==6.2.1
+# openpyxl==3.0.3
```

Comment:
Estos mejor borrarlos, no?

Reply:
Los he dejado comentados por si me equivoco y hay alguna parte de la web que los usa, y no este activa ahora. Yo esperaria a pasar aproduccion y estar un par de semanas, y luego ya quitarlos

Reply:
Para eso está git, kill it with 🔥